cobc/codegen.c

```diff
@@ -127,7 +127,6 @@ struct attr_list {
 struct literal_list {
 	struct literal_list	*next;
 	struct cb_literal	*literal;
-	cb_tree			x;
 	int			id;
 	int			make_decimal;
 };
```

Comment:
Shouldn't that struct be moved to tree.h?

Reply:
Actually I just added a forward declaration `struct literal_list;` in `tree.h`; that is sufficient (I don't see any reason to expose the whole type in `tree.h`).

Reply:
I confess that I don't know if that works like declarations of functions in ancient C compilers and also don't know how that looks like in GDB when inspecting the program. But we can test.

Reply:
You mean the program? Sure if that's possible without including tree.h in too many other sources - that would actually be preferable.

Reply:
I mean writing the full definition of `struct literal_list` in `tree.h` instead of just the forward declaration.